### DISCUSSION TITLE
Hosting

### DISCUSSION THREAD

**Discussion by georgkrause** [OpenPodcastAPI/api-specs#14] [General / Meta]
Hi everyone,

with #13 we now seem to have some need for a hosting solution.

I can offer to ask the Funkwhale Collective if we want to sponsor the hosting, I guess we have some space left on our server for some small things.

**Comment by keunes** [comment]
Thanks @georgkrause for the offer!

I'm thinking that for the docs we might use GitHub pages (?). Then we'd only need a small reference API server.

**Reply by georgkrause** [reply]
Well, thats an option. Maybe we should try not to be to much relying on ~~Gitlab~~ Github? But apart from that, I don't have any hard feelings.

Hosting the reference implementation is certainly not an issue.
